e2e/util_test.go

```diff
@@ -133,10 +160,35 @@ func setup(t *testing.T, options ...option) (*s3.S3, func(...string) icmd.Cmd) {
 	if opts.endpointURL != "" {
 		endpoint = opts.endpointURL
 	}
+	secretKey := ""
+	if opts.secretKey != "" {
+		secretKey = opts.secretKey
+	}
+
+	accessKeyID := ""
+	if opts.accessKeyID != "" {
+		accessKeyID = opts.accessKeyID
+	}
+
+	region := ""
+	if opts.region != "" {
+		region = opts.accessKeyID
+	}
+
+	var cfg *credentialCfg
+	if region != "" || accessKeyID != "" || secretKey != "" {
+		cfg = &credentialCfg{
+			AccessKeyID: accessKeyID,
+			SecretKey:   secretKey,
+			Region:      region,
+		}
+	} else {
+		cfg = nil
```

Comment:
Default value is already nil. don't need this else block.

Reply:
Resolved: [d1ff91e](https://github.com/peak/s5cmd/pull/611/commits/d1ff91e0db0af1179cd53a767e296e3c4297a6f4)